---
title: "Application of whole-exome sequencing for detecting copy number variants in CMT1A/HNPP"
collection: publications
permalink: /publication/2016-08-01-publication1
excerpt: 'Large insertions and deletions (indels), including copy number variations (CNVs), are commonly seen in many diseases. Standard approaches for indel detection rely on well-established methods such as qPCR or short tandem repeat (STR) markers. Recently, a number of tools for CNV detection based on next-generation sequencing (NGS) data have also been developed; however, use of these methods is limited. Here, we used whole-exome sequencing (WES) in patients previously diagnosed with CMT1A or HNPP using STR markers to evaluate the ability of WES to improve the clinical diagnosis. Patients were evaluated utilizing three CNV detection tools including CONIFER, ExomeCNV and CEQer, and array comparative genomic hybridization (aCGH). We identified a breakpoint region at 17p11.2-p12 in patients with CMT1A and HNPP. CNV detection levels were similar in both 6 Gb (mean read depth = 80×) and 17 Gb (mean read depth = 190×) data. Taken together, these data suggest that 6 Gb WES data are sufficient to reveal the genetic causes of various diseases and can be used to estimate single mutations, indels, and CNVs simultaneously. Furthermore, our data strongly indicate that CNV detection by NGS is a rapid and cost-effective method for clinical diagnosis of genetically heterogeneous disorders such as CMT neuropathy.'
date: 2016-08-01
venue: 'Clinical Genetics'
paperurl: 'http://hyeyeongJo.github.io/files/publication1.pdf'
citation: 'Hye-Yeong Jo et al.,. (2016). &quot;Application of whole-exome sequencing for detecting copy number variants in CMT1A/HNPP.&quot; <i>Journal 1</i>. 1(2).'
---
Large insertions and deletions (indels), including copy number variations (CNVs), are commonly seen in many diseases. Standard approaches for indel detection rely on well-established methods such as qPCR or short tandem repeat (STR) markers. Recently, a number of tools for CNV detection based on next-generation sequencing (NGS) data have also been developed; however, use of these methods is limited. Here, we used whole-exome sequencing (WES) in patients previously diagnosed with CMT1A or HNPP using STR markers to evaluate the ability of WES to improve the clinical diagnosis. Patients were evaluated utilizing three CNV detection tools including CONIFER, ExomeCNV and CEQer, and array comparative genomic hybridization (aCGH). We identified a breakpoint region at 17p11.2-p12 in patients with CMT1A and HNPP. CNV detection levels were similar in both 6 Gb (mean read depth = 80×) and 17 Gb (mean read depth = 190×) data. Taken together, these data suggest that 6 Gb WES data are sufficient to reveal the genetic causes of various diseases and can be used to estimate single mutations, indels, and CNVs simultaneously. Furthermore, our data strongly indicate that CNV detection by NGS is a rapid and cost-effective method for clinical diagnosis of genetically heterogeneous disorders such as CMT neuropathy.

[Download paper here](http://hyeyeongJo.github.io/files/publication1.pdf)

Recommended citation: Hye-Yeong Jo et al., (2016). "Application of whole-exome sequencing for detecting copy number variants in CMT1A/HNPP." <i>Clinical Genetics</i>. 1(2).
